app/gateway-oss/2.5.x/external-plugins.md

```diff
@@ -554,6 +555,28 @@ If your workload is CPU intensive, consider the multi-processing model by adding
 start_cmd.
 
 
+## Performance for external plugins
+
+Depending on implementation details, Go plugins are able to utilize multiple CPU cores
+and thus has best performance on a multi-core system. JavaScript plugins are currently
+single-core only and there's no dedicated plugin server support.
+Python plugins can use dedicated plugin server to span workload to
+multiple CPU cores as well.
+
+Unlike Lua plugins where invoking PDK functions are handled in local processes,
+calling PDK functions in external plugin implies inter-process communications and is thus
+relatively expensive operation. In that case the performance of Kong using external plugins is
```

Comment:
```suggestion
relatively expensive operation. Because of the expense of calling PDK functions in external plugins, the performance of Kong using external plugins is
```